functional/index.md

```diff
@@ -7,7 +7,7 @@ Rust is an imperative language, but it follows many functional programming parad
 ```rust
 let mut sum = 0;
 for i in 1..11 {
-	sum += i;
+ sum += i;
```

Comment:
Could we exclude code blocks from the linting? We should do that with `cargo fmt`, I've opened an upstream issue for that?